Quick_Deploy/vLLM/README.md

```diff
@@ -84,17 +55,18 @@ and
 [EngineArgs](https://github.com/vllm-project/vllm/blob/32b6816e556f69f1672085a6267e8516bcb8e622/vllm/engine/arg_utils.py#L11)
 for supported key-value pairs.
 
-*Note*: vLLM greedily consume upto 90% of the GPU's memory under default settings. You can provide
+*Note*: vLLM greedily consumes up to 50% of the GPU's memory under default settings. You can tweak this behavior using
 appropriate fields like `gpu_memory_utilization` and other settings via
 [`vllm_engine_args.json`](model_repository/vllm/vllm_engine_args.json).
 
 Read through the documentation in [`model.py`](model_repository/vllm/1/model.py) to understand how
 to configure this sample for your use-case.
 
-Using the container you already started, run the following command:
+Run the following commands to build the image and start the server container:
 
 ```
-tritonserver --model-store=model_repository
+docker build -t triton_vllm .
+docker run --gpus all -it --rm -p 8001:8001 --shm-size=1G --ulimit memlock=-1 --ulimit stack=67108864 -v ${PWD}:/work -w /work triton_vllm tritonserver --model-store ./model_repository
```

Comment:
Lowering to `--shm-size=1G` requirement is nice, more in line with requirements for some other popular examples, such as TGI:
```
docker run --gpus all --shm-size 1g -p 8080:80 -v $volume:/data ghcr.io/huggingface/text-generation-inference:1.0.3 --model-id $model
```